### ISSUE TITLE
Analyze top exceptions

### ISSUE THREAD

**Issue by pnadolny13** [meltano/squared#492]
Related to https://github.com/meltano/meltano/discussions/7016#discussion-4591592 we should see what exception data we have and if we can narrow in on the top 10.

It's challenging from a brief look because theyre hashed exceptions but it might be easier to unhash once we know the top values and maybe simulating a few exceptions to see how the exception translates to event attributes.

**Comment by aaronsteers** [comment]
> hashed exceptions

I thought we were keeping exception class names unhashed but hashing the message text. Can you confirm if you can at least see the class name in clear text, e.g. `'ValueError'` or `'DivideByZeroError'`?

**Comment by pnadolny13** [comment]
@aaronsteers yeah it looks like those are unhashed so if I'm following it correctly I see `PluginExecutionError` -> `RunnerError` -> `CliError` -> `SystemExit`. I dont know a lot about Meltano exceptions but I figured the actual exception is down in the plugin so I cant know if this a user error like "missing x required config" or a plugin error like "tap exploded because of a bug".

Let me know if theres a way for me to get at this data better but from a quick look it didnt seem obvious.
